pumps/pump.go

```diff
@@ -28,3 +34,19 @@ func GetPumpByName(name string) (Pump, error) {
 
 	return nil, errors.New(name + " Not found")
 }
+
+func processPumpEnvVars(pump Pump, log *logrus.Entry, cfg interface{}, defaultEnv string) {
+	if envVar := pump.GetEnvPrefix(); envVar != "" {
+		log.Debug(fmt.Sprintf("Checking %v env variables with prefix %v", pump.GetName(), envVar))
```

Comment:
```suggestion
		log.Debug(fmt.Sprintf("Checking %s env variables with prefix %s", pump.GetName(), envVar))
```
envVar and name are strings